tests/test_density_LocalDensity.py

```diff
@@ -65,6 +75,24 @@ def test_refpoints(self):
     def test_repr(self):
         self.assertEqual(str(self.ld), str(eval(repr(self.ld))))
 
+    def test_ref_point_ne_point(self):
+        box = freud.box.Box.cube(10)
+        ref_points = np.array([[0, 0, 0], [1, 0, 0]])
+        points = np.array([[0, 1, 0], [-1, -1, 0]])
+        volume = 1
+        diameter = 1
+        rcut = 2
+
+        v_around = 4/3 * (rcut**3) * np.pi
+
+        ld = freud.density.LocalDensity(rcut, volume, diameter)
+        ld.compute(box, ref_points, points)
+
+        cd0 = 2/v_around
+        cd1 = (1 + getFraction(np.sqrt(5), rcut, diameter))/v_around
```

Comment:
Can you replace `np.sqrt(5)` with `np.linalg.norm(ref_points[1] - points[1])`? Otherwise the 5 is a bit of a magic number.